rclpy/rclpy/node.py

```diff
@@ -247,6 +247,8 @@ def destroy_timer(self, timer):
         for tmr in self.timers:
             if tmr.timer_handle == timer.timer_handle:
                 _rclpy.rclpy_destroy_entity(tmr.timer_handle)
+                # TODO(sloretz) Store clocks on node and destroy them separately
+                _rclpy.rclpy_destroy_entity(tmr._clock._clock_handle)
```

Comment:
Instead of accessing a "protected" member (`tmr._clock`) should this be exposed through a getter?

Reply:
https://github.com/ros2/rclpy/pull/212/commits/d4c43c63ecedb30ffe6de7888f13c14f99a7533e